skada/tests/test_subspace.py

```diff
@@ -52,6 +59,9 @@ def test_subspace_alignment(estimator, da_dataset):
         (TransferComponentAnalysisAdapter(), 5, 3, 3),
         (TransferComponentAnalysisAdapter(), 2, 3, 3),
         (TransferComponentAnalysisAdapter(), 2, 5, 4),
+        (TransferJointMatchingAdapter(), 5, 3, 3),
```

Comment:
This test is to try if when we use `n_component=None` we use the good value, so for you, it is ` n_components = min(min(X_source.shape), min(X_target.shape))`.

Reply:
oui, donc 3 ici, non? (n_sample>n_feature), et on a 5 et 3 comme features, donc c'est 3